### PR TITLE
Future features tidiness

### PR DESCRIPTION
**Pull request by sunfishcode**
This moves #247 and #250 out of issues and into FutureFeatures.md where there is still no guarantee they'll be implemented, but we can close the issues. And it updates the FutureFeatures.md operator names in general.

### CODE REVIEW

File: AstSemantics.md [hunk 1]
```diff
@@ -375,7 +375,7 @@ results into the result type.
   * `int32.sub`: sign-agnostic subtraction
   * `int32.mul`: sign-agnostic multiplication (lower 32-bits)
   * `int32.div_s`: signed division (result is truncated toward zero)
-  * `int32.div_u`: unsigned division
+  * `int32.div_u`: unsigned division (result is floored)
```
Comment (jfbastien) [inline AstSemantics.md]: Is "floor" more correct than "truncated toward zero" for unsigned? The later is more familiar to mere mortals IMO.

File: FutureFeatures.md [hunk 2]
```diff
@@ -213,19 +213,25 @@ use cases:
...
-  * `int32.abs`: absolute value (is `abs(INT32_MIN)` `INT32_MIN` or should it trap?)
-  * `int32.bswap`: reverse bytes (endian conversion)
-  * `int32.bswap16`: `bswap16(x)` is `((x>>8)&255)|((x&255)<<8)`
+  * `int32.rotr`: sign-agnostic bitwise rotate right
+  * `int32.rotl`: sign-agnostic bitwise rotate left
+  * `int32.min_s`: signed minimum
+  * `int32.max_s`: signed maximum
+  * `int32.min_u`: unsigned minimum
+  * `int32.max_u`: unsigned maximum
+  * `int32.sext`: sign-agnostic `sext(x, y)` is `shr(shl(x,y),y)`
```
Comment (jfbastien) [inline FutureFeatures.md]: Use `shr_u` since there's no `shr` op? Or `_s`, since it doesn't matter. Something valid anyways :-)

Comment (jfbastien) [inline FutureFeatures.md]: Or follow the lead of `bswap16` below, and use `>>`.

File: FutureFeatures.md [hunk 3]
```diff
@@ -213,19 +213,25 @@ use cases:
...
+  * `int32.abs_s`: signed absolute value (traps on `INT32_MIN`)
+  * `int32.bswap`: sign-agnostic reverse bytes (endian conversion)
+  * `int32.bswap16`: sign-agnostic, `bswap16(x)` is `((x>>8)&255)|((x&255)<<8)`
 
 * The following operations are just potentially interesting.
-  * `int32.clrs`: count leading redundant sign bits (defined for all values, including 0)
+  * `int32.clrs`: sign-agnostic count leading redundant sign bits (defined for
+    all values, including 0)
+  * `int32.floor_div_s`: signed division (result is floored)
```
Comment (jfbastien) [inline FutureFeatures.md]: Floored is toward `-inf`?

File: FutureFeatures.md [hunk 4]
```diff
@@ -213,19 +213,25 @@ use cases:
...
+
+* The following 64-bit-only operations are potentially interesting as well.
+  * `int64.mor`: sign-agnostic 8x8 bit-matrix multiply with or
+  * `int64.mxor`: sign-agnostic 8x8 bit-matrix multiply with xor
```
Comment (jfbastien) [inline FutureFeatures.md]: I don't understand these without asking the Googles for help. Could you clarify?

File: AstSemantics.md (hunk 1 above)
Comment (sunfishcode) [inline AstSemantics.md]: They're equivalent for unsigned types, because there are no negative values. Floor is a common term though; I'll add a link.

**Comment by sunfishcode** [comment]
@jfb I added links to clarify the terms.

**Comment by jfbastien** [comment]
lgtm

**Comment by sunfishcode** [comment]
Merging based on lgtm above, and because this is FutureFeatures territory, which is of course wide open for future modification.

**Pull request merged**